src/test/java/net/masterthought/cucumber/json/StepTest.java

```diff
@@ -65,10 +65,10 @@ public void getOutput_ReturnsOutput() {
         Step step = features.get(1).getElements()[0].getSteps()[7];
 
         // when
-        Output output = step.getOutput();
+        Output[] outputs = step.getOutputs();
 
         // then
-        assertThat(output.getMessages()).containsOnlyOnce(
+        assertThat(getMessages(outputs)).containsOnlyOnce(
```

Comment:
generally I'm OK with this feature as long as this does not generate more code that is needed...